src/common/containers/GameArea.js

```diff
@@ -373,6 +377,7 @@ export class GameArea extends Component {
   render() {
     return (
       <div
+        id="gameArea"
```

Comment:
[nit] @jacobnisnevich convinced me to use hyphens (`game-area`) in markup. Makes it easier to distinguish the difference between when you're in HTML land and when you're in JS land.

Reply:
Yeah I agree with this.

Reply:
I agree in general, but it probably makes more sense to have a separate PR changing all IDs/classes to `kebab-case`. I'll merge this as-is for now.